src/battle_ai_main.c

```diff
@@ -3063,8 +3069,54 @@ static s32 AI_DoubleBattle(u32 battlerAtk, u32 battlerDef, u32 move, s32 score)
     switch (effect)
     {
     case EFFECT_HELPING_HAND:
-        if (!hasPartner || !HasDamagingMove(battlerAtkPartner))
-            ADJUST_SCORE(-20);
+        if (!hasPartner || !HasDamagingMove(battlerAtkPartner)
+        || (aiData->partnerMove != MOVE_NONE && IsBattleMoveStatus(aiData->partnerMove))
```

Comment:
```suggestion
        if (!hasPartner 
         || !HasDamagingMove(battlerAtkPartner)
         || (aiData->partnerMove != MOVE_NONE && IsBattleMoveStatus(aiData->partnerMove))
```
once you break once, it makes more sense to break everything in the within the same chain